docker-compose.yml

```diff
@@ -15,23 +15,31 @@ services:
     depends_on:
       - postgres
 
+  # Create EE container
+  cuke-master:
+    image: registry2.itci.conjur.net/conjur-appliance-cuke-master:5.0-stable
```

Comment:
this name might change - can you ask in #infrastructure to find out what it will change to / what you should use here?

Reply:
feedback given from infra channel: changing the tag is to be a future discussion because it is so widely used